core/src/main/java/nl/nn/adapterframework/soap/SoapWrapper.java

```diff
@@ -101,7 +97,7 @@ public static SoapWrapper getInstance() throws ConfigurationException {
 		return self;
 	}
 
-	public void checkForSoapFault(String responseBody, Throwable nested) throws SenderException {
+	public void checkForSoapFault(Message responseBody, Throwable nested) throws SenderException {
 		String faultString = null;
```

Comment:
de responseBody **moet** hier persistent zijn, anders gaan hier fouten ontstaan. Misschien is het handig de tests aan te passen, aangezien dit niet getest wordt!